pramen-py/tests/test_runner/test_runner_transformation.py

```diff
@@ -0,0 +1,37 @@
+#  Copyright 2022 ABSA Group Limited
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+import pathlib
+
+from environs import Env
+
+from pramen_py.runner.runner_transformation import (
+    DEFAULT_TRANSFORMATIONS_DIR,
+    get_transformations_dir,
+)
+
+
+def test_get_transformations_dir():
+    os.environ["test_dir_os"] = "os"
+    with open(".env.for_test", "w") as fobj:
```

Comment:
pytest has [tmp_path](https://docs.pytest.org/en/7.2.x/how-to/tmp_path.html#the-tmp-path-fixture)